pipeline/docker/__init__.py

```diff
@@ -0,0 +1,81 @@
+from ntpath import join
+import os
+import yaml
+
+from typing import List
+
+from pipeline.objects import Graph
+
+
+def create_pipeline_api(
+    pipeline_graphs: List[Graph], *, output_dir="./", **environment_variables
+):
+    paths = []
+
+    for pipeline_graph in pipeline_graphs:
+        graph_path = pipeline_graph.name + ".graph"
+        pipeline_graph.save(graph_path)
+        paths.append(graph_path)
+
+    create_dockerfile(paths, output_dir=output_dir)
+    create_docker_compose(output_dir, **environment_variables)
+
+
+def create_dockerfile(pipeline_graph_paths: List[str], *, output_dir="./"):
+    with open(os.path.join(output_dir, "Dockerfile"), "w") as docker_file:
+        docker_file.writelines(
+            ["FROM --platform=linux/amd64 mysticai/pipeline-docker:latest"]
```

Comment:
Is `--platform=linux/amd64` necessary? Can we not provide native containers? 😢

Reply:
It's mostly for consistency, building on M1 will not work in some occasions or it can't then be passed on.
We can add this as a kwarg to the dockerize function so that people can select what OS they want.